api/cpp/CMakeLists.txt

```
@@ -90,6 +90,8 @@ define_cargo_feature(renderer-winit-skia-opengl "Enable support for the Skia bas
 
 define_cargo_feature(backend-qt "Enable Qt based rendering backend" ON)
 
+define_cargo_feature(experimental "Enable experimental features (no compatibility guarentees" OFF)
```

It's a bit of a weird name for a feature - the "what" is missing. The feature itself is not experimental, it enables other experimental features. How about calling it `SLINT_FEATURE_ENABLE_EXPERIMENTAL_FEATURES`? Then the define is also nicer, as `SLINT_FEATURE_EXPERIMENTAL` sounds like just one experimental feature.

Another option would be `SLINT_FEATURE_EXPERIMENTAL_NAMESPACE`.

```suggestion
define_cargo_feature(experimental "Enable experimental features (no compatibility guarantees)" OFF)
```